### PR TITLE
thors-serializer 3.2.6

### PR DESCRIPTION
**Pull request by Loki-Astari**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by carlocab** [comment]
```
  In file included from BsonParser.cpp:1:
  In file included from /private/tmp/thors-serializer-20240804-10525-1rd0yu/src/Serialize/BsonParser.h:21:
  In file included from /private/tmp/thors-serializer-20240804-10525-1rd0yu/src/Serialize/Serialize.h:318:
  /private/tmp/thors-serializer-20240804-10525-1rd0yu/src/Serialize/Serialize.tpp:451:35: error: missing 'typename' prior to dependent type name 'Trait::RefType'
              using RefType       = Trait::RefType;
                                    ^~~~~~~~~~~~~~
                                    typename 
```
https://github.com/Homebrew/homebrew-core/actions/runs/10242010961/job/28332091833?pr=180026#step:3:2789

**Review by carlocab — APPROVED** [review]

**Review by bevanjkay — APPROVED** [review]

**Comment by carlocab** [comment]
Still the same error I think:
https://github.com/Homebrew/homebrew-core/actions/runs/10242731313/job/28332961305#step:3:2796

**Comment by Loki-Astari** [comment]
Sorry about that.
The output had been trucated so I just repaired the error you quoted. The first time.

I got the full error message this time and fixed all the issues (I hope) this go.

**Review by carlocab — APPROVED** [review]
Thanks!

File: Formula/t/thors-serializer.rb [hunk 1]
```diff
@@ -29,7 +30,9 @@ def install
 
     system "./configure", "--disable-vera",
                           "--prefix=#{prefix}",
-                          "--disable-test-with-integration"
+                          "--disable-test-with-integration",
+                          "--disable-test-with-mongo-query",
+                          "--disable-Mongo-Service"
```
Comment (p-linnane) [inline Formula/t/thors-serializer.rb:35]:
```suggestion
                          "--disable-Mongo-Service",
                          *std_configure_args.reject { |s| s["--disable-debug"] }
```

Comment (Loki-Astari) [reply Formula/t/thors-serializer.rb:37]: This would not work with my config file.

**Review by chenrui333 — APPROVED** [review]

**Pull request merged**